tasks/sep/006-1-guardian-upgrade/.env

```diff
@@ -1,4 +1,4 @@
-ETH_RPC_URL=https://1rpc.io/sepolia	 # Fill this in with an L1 Sepolia RPC URL that has archive data access.
+ETH_RPC_URL=https://1rpc.io/sepolia
 OWNER_SAFE=0x1Eb2fFc903729a0F03966B917003800b145F56E2
 COUNCIL_SAFE=0xf64bc17485f0B4Ea5F06A96514182FC4cB561977
 FOUNDATION_SAFE=0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B
```

Comment:
Should the nonce be set?

Reply:
In this one we can't because all nonces differ and we only have a single env var for controlling the nonce

```sh
# Nonce 7 for OWNER_SAFE
cast call 0x1Eb2fFc903729a0F03966B917003800b145F56E2 "nonce()(uint)" -r $SEPOLIA_RPC_URL 

# Nonce 9 for COUNCIL_SAFE
cast call 0xf64bc17485f0B4Ea5F06A96514182FC4cB561977 "nonce()(uint)" -r $SEPOLIA_RPC_URL

# Nonce 19 for FOUNDATION_SAFE
cast call 0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B "nonce()(uint)" -r $SEPOLIA_RPC_URL
```